README.md

```diff
@@ -19,10 +19,29 @@ sets trasmitted invoice to paid in Alma
 - `SAP_FINAL_RECIPIENT_EMAIL` = moira list to recieves final run emails
 - `SAP_REVIEW_RECIPIENT_EMAIL` = moira list to recieve review run emails
 - `SES_SEND_FROM_EMAIL` = email address that SES sends from
-- `SSM_PATH` = the path to ssm variables 
+- `SAP_SEQUENCE_NUM` = the SSM path of the current SAP sequence number 
 - `WORKSPACE` = Set to `dev` for local development, this will be set to `stage` and `prod` in those environments by Terraform.
 
 ## Optional ENV
 - `ALMA_API_TIMEOUT`= Request timeout for Alma API calls. Defaults to 30 seconds if not set. 
 - `LOG_LEVEL` = Set to a valid Python logging level (e.g. DEBUG, case-insensitive) if desired. Can also be passed as an option directly to the ccslips command. Defaults to INFO if not set or passed to the command.
-- `SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
\ No newline at end of file
+- `SENTRY_DSN` = If set to a valid Sentry DSN, enables Sentry exception monitoring. This is not needed for local development.
+
+## Creating sample SAP data
+Running the SAP Invoices process during local development or on staging requires that
+there be sample invoices ready to be paid in the Alma sandbox. To simplify this, there
+is a CLI command that will create four sample invoices in the sandbox. To do this:
+  1. Run `pipenv run sap create-sandbox-data`. You should get a final log message
+     saying there are four invoices ready for manual approval in Alma.
+  2. Go to the Alma sandbox UI > Acquisitions module > Review (Invoice) > Unassigned
```

Comment:
This can be updated...I think we changed the process at some point so the Review step is no longer necessary, they just go straight to the Waiting For Approval queue.